miq_version/__init__.py

```diff
@@ -216,94 +216,65 @@ def get_version(obj=None):
 UPSTREAM = LATEST
 
 SPTuple = namedtuple('StreamProductTuple', ['stream', 'product_version', 'template_regex'])
-TemplateInfo = namedtuple('TemplateInfo', ['group_name', 'datestamp', 'stream', 'version'])
-
+TemplateInfo = namedtuple('TemplateInfo', ['group_name', 'datestamp', 'stream', 'version', 'type'])
+
+FORMATS_DOWNSTREAM = {
```

Comment:
*OPTIONAL* It'd be really helpful if there's a comment for each of these showing an example string that matches the regex